scripts/compile/compile-stage.ps1

```diff
@@ -59,6 +61,11 @@ $Projects | ForEach-Object {
     }
 }
 
+if (Test-Path $binariesOutputDir) {
+    cp $binariesOutputDir\* $OutputDir\bin -force -recurse
+    Remove-Item $binariesOutputDir -recurse
```

Comment:
I see what you are saying. The problem of leaving it here is that it would be packed out and shipped twice inside the zip/tar.gz and msi/pkg/deb, etc. I think the solution here is to change the publish output dir for the compile step to somewhere else outside the stage dir and then copy that over to Stage2Dir/bin and runtime.

What do you think?